service_worker_api/index.d.ts

```diff
@@ -576,12 +576,73 @@ interface ServiceWorkerRegistration extends EventTarget {
      * before it is unregistered.
      */
     unregister(): Promise<boolean>;
+
+    /**
+     * Returns a Promise that resolves to an array of Notification objects.
+     * @param [options] An options object that can contain options to filter the notifications returned.
+     * The available options are:
+     * tag: A DOMString representing a notification tag. If specified, only notifications that have this tag
+     * will be returned.
+     */
+    getNotifications(options?: ServiceWorkerGetNotificationOptions): Promise<any[]>; // need to be replaced with `Notification[]` when possible
+
+    /**
+     * Displays the notification with the requested title.
+     * @param [title] The title that must be shown within the notification
+     * @param [options] An object that allows to configure the notification. It can have the following properties:
+     * actions: An array of actions to display in the notification. The members of the array should be an object
```

Comment:
Can you transfer the descriptions of the properties to their declarations in the `ServiceWorkerNotificationOptions` interface?